sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/NamespaceChange.java

```diff
@@ -29,6 +31,7 @@
  *     )
  * </pre>
  */
+@Experimental
```

Comment:
I'm okay with this, but we should be more careful to make sure these annotations are included earlier in future PRs.